res/layout/overlay_button.xml

```diff
@@ -0,0 +1,19 @@
+<!-- overlay_button.xml -->
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal">
+
+    <Button
+        android:id="@+id/invokeKeyboardButton"
+        android:layout_width="72dp"
+        android:layout_height="wrap_content"
+        android:text="Show" />
```

Comment:
This buttons shows up on top of every other apps. "Show" doesn't mean much without context. The app's icon or a keyboard symbol would be more intuitive.